tsconfig.json

```
@@ -48,7 +48,7 @@
         "./system/*.ts"
       ],
       "#trainers/*": ["./data/trainers/*.ts"],
-      "#types/*": ["./@types/*.ts", "./typings/phaser/*.ts"],
+      "#types/*": ["./@types/*.ts", "./@types/helpers/*.ts", "./typings/phaser/*.ts"],
```

```suggestion
      "#types/*": ["./@types/helpers/*.ts", "./@types/*.ts", "./typings/phaser/*.ts"],
```
You need to put the deeper nesting paths first, because it's a "use first match" system.

---

No, `tsconfig.json` does not support that.